### ISSUE TITLE
Sources tab for Wget package results in 404

### ISSUE THREAD

**Issue by noloader** [fedora-infra/fedora-packages#401]
I'm trying to find the patches applied to Wget when Fedora builds the package. I am at [Fedora Packages | Wget](https://apps.fedoraproject.org/packages/wget). The Sources tab does not provide content (instead the previous tab is displayed).

When I right click the Sources tab and Open in a new Tab, it results in a 404:

![wget-package](https://user-images.githubusercontent.com/3538226/46838098-bcff6a00-cd85-11e8-9c02-08733b6224f0.png)

**Comment by cverna** [comment]
Currently the sources tab is not functional, There is a PR #390 opened to start replacing the current implementation.

**Comment by ferdnyc** [comment]
> [Issue 329, "Sources" tab is not working](https://github.com/fedora-infra/fedora-packages/issues/329) may be related. But it looks like the fix was pushed in January 2018, and this is probably a new issue (or maybe not).

Nope, same old bug. That was reported as "fixed in staging", with said fix to be imminently released, back in February. ...And that was the last anyone ever heard of said fix.

What _hasn't_ been mentioned though, in terms of your actual goal (and for the edification of any future readers of this bug), is that the sources to whatever project you're viewing **can** still be accessed from Packages. 

* Choose "SCM" from the "$PROJECT in other apps" dropdown beside its name:

   ![screenshot from 2018-11-05 13-27-22](https://user-images.githubusercontent.com/538020/48018549-1ba6d200-e0ff-11e8-8817-d53cb25c2afc.png)

* That'll take you to the snazzy new Package Sources server, far nicer than crappy cgit:

   ![screenshot from 2018-11-05 13-28-43](https://user-images.githubusercontent.com/538020/48018580-31b49280-e0ff-11e8-8845-46fe2ac28078.png)

   (Same thing the Sources tab of Packages is _meant_ to show.)

...And in this case, that's what you were looking for. 

* OTOH, if you wanted to reach the _upstream_ sources, there's a link that shows (only) on the Overview tab of Package Sources project pages, next to the summary line below the project name:

   ![screenshot from 2018-11-05 13-28-54](https://user-images.githubusercontent.com/538020/48018677-76d8c480-e0ff-11e8-8be6-f0fc5674e185.png)
